src/components/select-field/SelectDropdownList.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+
+import type { SelectOptionValueProp } from './props';
+
+export const OVERLAY_SCROLLABLE_CLASS = 'bdl-SelectField-overlay--scrollable';
+
+type Props = {
+    children: React.Node,
+    isScrollable?: boolean,
+    multiple?: boolean,
+    placement: string,
+    popperRef: (?HTMLElement) => void,
+    scheduleUpdate?: () => void,
+    selectFieldID: String,
+    selectedValues: Array<SelectOptionValueProp>,
+    style: Object,
+};
+
+class SelectDropdownList extends React.Component<Props> {
+    componentDidUpdate({ selectedValues: prevSelectedValues }) {
+        const { scheduleUpdate, selectedValues } = this.props;
+        if (scheduleUpdate && prevSelectedValues !== selectedValues) {
+            scheduleUpdate();
+        }
+    }
+
+    render() {
+        const { children, popperRef, style, placement, isScrollable, multiple, selectFieldID } = this.props;
+
+        const listboxProps = {};
+        if (multiple) {
+            listboxProps['aria-multiselectable'] = true;
+        }
+
+        return (
+            <ul
+                ref={popperRef}
```

Comment:
Can we use `React.forwardRef` in this class, as well, to avoid this renaming?

Reply:
I kinda like using named ref (innerRef etc...) more than forwardRef because the flow types are so hard to read/understand and people just end up dumping <any>